Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardItemRow.swift

```diff
@@ -78,17 +76,14 @@ struct CreditCardItemRow: View {
             .padding(.trailing, 16)
             .padding(.top, 11)
             .padding(.bottom, 11)
-            .background(isTapped ? backgroundHoverColor : backgroundColor)
-            .gesture(
-                TapGesture()
-                    .onEnded { _ in
-                        didSelectAction?()
-                    }
-                    .simultaneously(with: DragGesture(minimumDistance: 0))
-                    .updating($isTapped) { (_, isTapped, _) in
-                        isTapped = true
-                    }
-            )
+            .background(isTapping ? backgroundHoverColor : backgroundColor)
+            .onTapGesture {
+                isTapping = true
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                    isTapping = false
+                    didSelectAction?()
+                }
+            }
```

Comment:
This feels a little arbitrary, there must be a better way of managing gestures in SwiftUI. My lack of SwiftUI experience isn't going to help here so handing it off to @nbhasin2 to review.

Reply:
Ah I knew this would come to bite me, sometimes things are too good to be true (Thanks SwiftUI) 

Let me take a look and see if there is a better option here 🤞🏼

Reply:
There doesn't exist a 1:1 mapping of touch events of UIKit inside SwiftUI so a workaround for this is something you have found @petruSt24 

There are 2 variations, I am leaning towards `Variation A.` to keep it simple but I also like `Variation B.`
This is more of UX question at this point. 

**Variation A.**
This is the same as your proposal but with `isTapped = false` removed as you are already dismissing the view so leaving the selection shows the user about the card they tapped on even if it's for split second.

```
            .onTapGesture {
                isTapping = true
                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
                    didSelectAction?()
                }
            }
```

**Variation B.** 
This is similar to what you have but has two gestures one is for dragging and another one for tap. This should not interfere with Scroll Gesture unless user spends way too much time holding the finger on the card at that point the card will get highlighted. 

Likes: I like it because it allows scrolling but if user is just holding their finger they know the card they have selected remains highlighted. 

Dislike: Its overkill for the problem we are trying to solve 🤷🏼‍♂️ and don't really need it.

You will need to update the following vars and the code below to make this work

```
    @State private var isTapped = false
    @GestureState var isDragging = false 
 ```
 
```
            .background(isTapped || isDragging ? backgroundHoverColor : backgroundColor)
            .gesture(
                TapGesture()
                    .onEnded { _ in
                        isTapped = true
                        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
                            didSelectAction?()
                        }
                    }
                    .exclusively(before: DragGesture(minimumDistance: 0))
                    .updating($isDragging) { (_, isDragging, _) in
                        isDragging = true
                    }
            )
```


Conclusion: Go with `Variation A.` and if UX likes the other version we can easily update it. But these are the best solutions for a small UX issue. The long solution would be creating a special HostingTableViewCell only for this view which isn't something I am keen on doing.